<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtaWidget</class>
 <widget class="QWidget" name="OtaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>服务器回复</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="let_dns"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>ota文件</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>服务器ip</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPlainTextEdit" name="pte_serderInfo"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>域名</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>客户端请求信息</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="let_serverIp"/>
   </item>
   <item row="4" column="2">
    <widget class="QPlainTextEdit" name="pte_clientInfo"/>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pbt_otaFileLoad">
     <property name="text">
      <string>load</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="let_otaFile"/>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pbt_dnsSet">
     <property name="text">
      <string>生效</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
